pkg/node/node.go

```diff
@@ -193,30 +200,26 @@ func NewBee(addr string, swarmAddress swarm.Address, publicKey ecdsa.PublicKey,
 
 	addressbook := addressbook.New(stateStore)
 
-	var swapBackend *ethclient.Client
-	var overlayEthAddress common.Address
-	var chainID int64
-	var transactionService transaction.Service
-	var transactionMonitor transaction.Monitor
 	var chequebookFactory chequebook.Factory
 	var chequebookService chequebook.Service
 	var chequeStore chequebook.ChequeStore
 	var cashoutService chequebook.CashoutService
 
-	if o.SwapEnable {
-		swapBackend, overlayEthAddress, chainID, transactionMonitor, transactionService, err = InitChain(
-			p2pCtx,
-			logger,
-			stateStore,
-			o.SwapEndpoint,
-			signer,
-		)
-		if err != nil {
-			return nil, err
-		}
-		b.ethClientCloser = swapBackend.Close
-		b.transactionMonitorCloser = transactionMonitor
+	swapBackend, overlayEthAddress, chainID, transactionMonitor, transactionService, err := InitChain(
```

Comment:
maybe this should be in an `if !o.Standalone`, otherwise standalone mode will require an eth backend.